docs/edit_ldap_module.rst

```diff
@@ -433,7 +433,7 @@ Examples
           encry: "SSL"
           address: "100.2.2.2"
           server_port: 389
-          dn: "cn=manager,ou=login,dc=domain,dc=com"
+          dn: "cn=manager, ou=login, dc=domain, dc=com"
```

Comment:
@ieisystem Hi, just looking through this again and noticed the additional spaces might cause an issue with this LDAP expression. Should we create a new PR to fix it?

Reply:
Yes, Spaces cause an LDAP feature setup exception. I will fix this problem, there is no need to open a new PR. Thank you for checking. Thanks!